build-aux/main.mk

```diff
@@ -229,13 +229,27 @@ format: $(tools/golangci-lint) $(tools/protolint) ## (QA) Automatically fix lint
 	$(tools/golangci-lint) run --fix --timeout 2m ./... || true
 	$(tools/protolint) lint --fix rpc || true
 
-.PHONY: check
+.PHONY: check-all
```

Comment:
Looks like the name of the target below isn't renamed to `check-all`

Reply:
oops good catch thank you!